client/my-sites/plans-features-main/test/index.jsx

```diff
@@ -4,6 +4,7 @@
 
 jest.mock( 'calypso/components/marketing-message', () => () => null );
 jest.mock( '@automattic/plans-grid-next', () => ( {
+	...jest.requireActual( '@automattic/plans-grid-next' ),
```

Comment:
The tests now also need access to `usePlansForFeaturesGrid` and `usePlansForComparisonGrid`.